WordPress/src/main/res/values/dimens.xml

```diff
@@ -329,7 +329,6 @@
     <dimen name="margin_text_input_layout_baseline">20dp</dimen>
     <dimen name="margin_extra_extra_extra_large">92dp</dimen>
     <dimen name="promo_indicator_bullet_size">4dp</dimen>
-    <dimen name="textinputlayout_baseline_correction">2dp</dimen>
```

Comment:
Can we also move `textinputlayout_correction_padding` and `textinputlayout_correction_margin_right`. Those are part of the same set of "hacks" to fix issues of the TextInputLayout an pretty much are used in tandem. WDYT?

Reply:
Good point, done in 1d6f478.